docs/griptape-framework/structures/task-memory.md

```diff
@@ -1,89 +1,320 @@
 ## Overview
-Task Memory augments activity inputs and outputs with storage capabilities. It's a way to keep any data generated by tools off prompt but still allow LLMs to operate on that data remotely. This is useful in the following scenarios:
+
+Task Memory is a powerful feature of Griptape that allows you to control where the data returned by Tools is stored. This is useful in the following scenarios:
```

Comment:
How does "the data returned by Tools" relate to the data returned by the LLM? It may be useful to add context in relation to the model as that is what most new users will be familiar with.